taskcoordinator/app/src/test/java/com/microservices/taskcoordinator/controllers/CoordinatorControllerIntegrationTests.java

```diff
@@ -77,12 +85,113 @@ public void testCoordinateOrder_allValid_orderCoordinatedAndLaundryStateChanged(
         OrderDto orderCreated = orderService.getOrderById(INPUT_ORDER_ID);
         LaundryStateDto laundryStateForCreatedOrder = laundryStateService.getLaundryStateById(orderSubmissionDto.getLaundryId());
 
-        Assert.assertEquals(coordinationDto.getOrderId(), orderSubmissionDto.getOrderId());
-        Assert.assertEquals(orderSubmissionDto.getOrderId(), orderCreated.getId());
-        Assert.assertEquals(OrderStatus.RESERVED, orderCreated.getStatus());
+        assertEquals(coordinationDto.getOrderId(), orderSubmissionDto.getOrderId());
+        assertEquals(orderSubmissionDto.getOrderId(), orderCreated.getId());
+        assertEquals(OrderStatus.RESERVED, orderCreated.getStatus());
 
         long reserveDiff = laundryStateForCreatedOrder.getReservedTime() - laundryWillBeChosen.getReservedTime();
-        Assert.assertEquals((long) orderCreated.getDuration(), reserveDiff);
+        assertEquals((long) orderCreated.getDuration(), reserveDiff);
     }
 
+    @Test
+    @Sql(scripts = "/test-data/one_laundry.sql")
+    @Transactional
+    public void testSubmitOrder_fromReservedStatus() throws Exception {
+        int orderId = 1;
```

Comment:
use constants like i did in previous test (like EXISTING_ORDER_ID or RESERVED_ORDER_ID)